RELEASE_NOTES.rst

```diff
@@ -1,3 +1,88 @@
+coala-bears 0.10.0
+==================
+
+This release, we had 97 commits contributed by 42 contributors
+
+We have updated the coala website to provide a quick overview of bears and
+their settings. To access this, visit http://coala.io/#!/languages. We also
```

Comment:
Hmm.. maybe we should update coala.io/languages so that it will redirect to this http://coala.io/#!/languages.
Then this note should just refer to coala.io/languages.
Thoughts?